recipes-omnect/images/omnect-os-image.bb

```diff
@@ -43,13 +43,12 @@ IMAGE_INSTALL = "\
     polkit \
     u-boot-fw-utils \
     systemd-analyze \
+    e2fsprogs-tune2fs \
+    jq \
+    smartmontools \
```

Comment:
imho remove it for now ... e.g. we can discuss if we add it for devices with non emmc storage
@JanZachmann?

Reply:
@mlilien you only talk about smartmontools correct? yes, let's remove and maybe bring back in a container someday

Reply:
done